File: fsharp-giraffe-server/OpenAPI/src/model/MissingArgumentErrorAllOf.fs
namespace OpenAPI.Model

open System
open System.Collections.Generic
open OpenAPI.Model.AnyType

module MissingArgumentErrorAllOf = 

  //#region MissingArgumentErrorAllOf


  type MissingArgumentError_allOf = {
    Result : AnyType;
    Msg : AnyType;
    Code : AnyType;
    VarName : string;
  }
  //#endregion
  

File: scala-gatling/src/gatling/scala/org/openapitools/client/model/MissingArgumentErrorAllOf.scala

package org.openapitools.client.model


case class MissingArgumentErrorAllOf (
    _result: Option[AnyType],
    _msg: Option[AnyType],
    _code: Option[AnyType],
    /* It contains the information about the missing parameter.  */
    _varName: Option[String]
)
object MissingArgumentErrorAllOf {
    def toStringBody(var_result: Object, var_msg: Object, var_code: Object, var_varName: Object) =
        s"""
        | {
        | "result":$var_result,"msg":$var_msg,"code":$var_code,"varName":$var_varName
        | }
        """.stripMargin
}
